src/rust/ide/src/controller/searcher.rs

```diff
@@ -693,7 +777,7 @@ impl Searcher {
         }).collect()
     }
 
-    fn get_suggestion_list_from_engine
+    fn schedule_gathering_action_list
```

Comment:
I dont understand the name of the function. Maybe its ok with docs, but now its mysterious